jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -375,22 +426,204 @@ public int hashCode() {
         }
     }
 
+    public static class TieredBlockJson {
```

Comment:
Are you on JDK8? Could you switch to JDK7 and retry to build it?

See https://github.com/antlr/antlr4/issues/494.